src/main.rs

```diff
@@ -175,15 +175,13 @@ fn handle_docs(cli: Cli) -> Result<(), Box<dyn Error>> {
     }
 }
 
-/*
-#[cfg(target_os = "windows")]
+#[cfg(windows)]
```

Comment:
If we have a conditional build for Windows (even if not officially supported), surely it shouldn't be commented out?

Reply:
See #490.